convert_types.go

```diff
@@ -22,26 +22,27 @@ import (
 // AnyPtr is a helper routine that allocates a new interface value
 // to store v and returns a pointer to it.
 //
-// 	// Usage: var _ *Type = AnyPtr(Type(value) | value).(*Type)
+// Usage: var _ *Type = AnyPtr(Type(value) | value).(*Type)
 //
-// 	var _ *bool = AnyPtr(true).(*bool)
```

Comment:
this is intentionally indented with an extra couple of spaces to have the editor integration show it as an example. we'll need to revert this as this breaks the example blocks.

![](https://shares.jacobbednarz.com/FQUBloV9-F9CHTeJ4-ZVr5S1Tx.png)

Reply:
the problem was that this is not considered formatted by go fmt rules, running `go fmt ./...` will reformat it